skimage/future/graph/rag.py

```diff
@@ -51,6 +51,67 @@ def min_weight(graph, src, dst, n):
     return min(w1, w2)
 
 
+def mean_color(graph, extra_arguments=[], extra_keywords={}):
+    """Callback to compute edge weights.
+
+    The method expects that each node has already been described.
+
+    Parameters
+    ----------
+    graph : RAG
+        The graph under consideration.
+    extra_arguments : sequence, optional
+        Allows extra positional arguments passed.
+    extra_keywords : dictionary, optional
+        Allows extra keyword arguments passed.
+        sigma = 255
+        mode = 'distance'
+    """
+    sigma = extra_keywords['sigma']
+    mode = extra_keywords['mode']
+    for x, y, d in graph.edges_iter(data=True):
+        diff = graph.node[x]['mean color'] - graph.node[y]['mean color']
+        diff = np.linalg.norm(diff)
+        if mode == 'similarity':
+            d['weight'] = math.e ** (-(diff ** 2) / sigma)
+        elif mode == 'distance':
+            d['weight'] = diff
+        else:
+            raise ValueError("The mode '%s' is not recognised" % mode)
+
+
+def describe_node(graph,labels,image, *args, **kwargs):
```

Comment:
Agreed.  Function belongs in an example of how to use `rag_generic`, rather than this module.  Included in PR to demonstrate how `rag_generic` can provide the same functionality as `rag_mean_color`.  Will remove and generate some example code.